Modules/_sqlite/connection.c

```diff
@@ -1478,6 +1478,94 @@ pysqlite_connection_iterdump(pysqlite_Connection* self, PyObject* args)
 }
 
 static PyObject *
+pysqlite_connection_backup(pysqlite_Connection* self, PyObject* args, PyObject* kwds)
```

Comment:
Ok, is it better to omit the whole function or rather raise an `NotImplementedError`?